docker/services/web/pytest/entrypoint.pytest.sh

```diff
@@ -3,8 +3,11 @@
 echo "sleeping entrypoint"
 sleep 120
 
+# python
+flask run --host=0.0.0.0
+
 # FIXME: unable to run pytest in parallel with xdist plugin due to the error
 #  Transaction (Process ID 52) was deadlocked on lock resources with another process and has been chosen as the deadlock victim. Rerun the transaction. (1205) (SQLExecDirectW)')
 # pytest -n auto --cipdb
 
-pytest --cipdb
\ No newline at end of file
+# pytest --cipdb
```

Comment:
commenting on everything is a bad practice. Just remove all comments if no longer needed